### PR TITLE
Add support for arguments of type Closure

### PR DESCRIPTION
**Pull request by alnivv-vi**
fix error for executeInSelenium function (Closure as argument)

fixes https://github.com/allure-framework/allure-codeception/issues/109

### CODE REVIEW

**Comment by baev** [comment]
@alnivv-vi, thanks for the PR. Could you please fix the build?

**Comment by alnivv-vi** [comment]
@baev [check](https://github.com/allure-framework/allure-codeception/actions/runs/4014274373) please

File: src/Internal/ArgumentAsString.php [hunk 1]
```diff
@@ -31,7 +31,7 @@ public function __construct(
 
     public static function get(mixed $argument): string
     {
-        return (string) new self($argument);
+        return (string)new self($argument);
```
Comment (baev) [inline src/Internal/ArgumentAsString.php:34]: revert

File: src/Internal/ArgumentAsString.php [hunk 2]
```diff
@@ -56,26 +56,30 @@ private function prepareString(string $argument): string
...
+
     private function prepareObject(object $argument): string
     {
-        if (isset($argument->__mocked) && is_object($argument->__mocked)) {
+        if (!$this->isClosure($argument) && isset($argument->__mocked) && is_object($argument->__mocked)) {
             $argument = $argument->__mocked;
         }
         if ($argument instanceof Stringable) {
-            return (string) $argument;
+            return (string)$argument;
```
Comment (baev) [inline src/Internal/ArgumentAsString.php:81]: revert

**Review by baev — CHANGES_REQUESTED** [review]

**Comment by baev** [comment]
@alnivv-vi thanks!

**Pull request merged**